server/protocols/protocols-imap4/src/main/java/org/apache/james/imapserver/netty/ImapChannelUpstreamHandler.java

```diff
@@ -117,6 +126,7 @@ public void channelConnected(ChannelHandlerContext ctx, ChannelStateEvent e) thr
         
         InetSocketAddress address = (InetSocketAddress) ctx.getChannel().getRemoteAddress();
         getLogger(ctx.getChannel()).info("Connection established from " + address.getAddress().getHostAddress());
+        imapConnectionsMetric.mark();

```

Comment:
I assume this metric indicates the number of connexions that has been established since the launch of the server.
Can you have a metric about current open connexions? So you'll need something like unmarm() or increase()/decrease()...